docs/app/documentation/component-docs/date-picker/examples/date-picker-i18n-example.component.ts

```diff
@@ -70,7 +104,26 @@ export class CustomI18nLabels extends CalendarI18nLabels {
     ]
 })
 export class DatePickerI18nExampleComponent {
+    languageService: LanguageService;
+    selected: number = 3;
+
+    constructor(languageService: LanguageService) {
+        this.languageService = languageService;
```

Comment:
This should usually never be in an angular app, you should provide the service instead